app/src/components/inputs/inputFilter/inputFilterToolbar/inputFilterToolbar.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import classNames from 'classnames/bind';
+import { BigButton } from 'components/buttons/bigButton';
+import styles from './inputFilterToolbar.scss';
+
+const cx = classNames.bind(styles);
+
+export class InputFilterToolbar extends PureComponent {
+  static propTypes = {
+    onApply: PropTypes.func,
+    onCancel: PropTypes.func,
+    onClear: PropTypes.func,
+  };
+  static defaultProps = {
+    onApply: () => {},
+    onCancel: () => {},
+    onClear: () => {},
+  };
+
+  render() {
+    const { onApply, onClear, onCancel } = this.props;
+
+    return (
+      <div className={cx('toolbar')}>
```

Comment:
Use `className` for component wrapper according to component's name but in dash notation: `input-filter-toolbar`